src/Lumi/Styles/QRCode.purs

```diff
@@ -0,0 +1,16 @@
+module Lumi.Styles.QRCode where
+
+import Prelude
+import Lumi.Components (PropsModifier)
+import Lumi.Styles (styleModifier_)
+import Lumi.Styles.Box (box)
+import React.Basic.Emotion (css, str)
+
+qrcode :: forall props. PropsModifier props
+qrcode =
+  box
+    >>> styleModifier_
+        ( css
+            { label: str "qrcode"
```

Comment:
What does this do? I don't recall a CSS attribute named `label`.

Reply:
It puts `qrcode` in the generated class name

Reply:
<img width="478" alt="image" src="https://user-images.githubusercontent.com/715921/75914605-de914380-5e09-11ea-8020-766505dc6718.png">

Reply:
Ah interesting! Is this an Emotion thing then? It might be worth having that as a separate style combinator in the future. It seems pretty useful.

Reply:
Yep